### PR TITLE
Issue#1063 - home mouthwash collection query and 5 day lag

### PR DESCRIPTION
**Pull request by Gbarra9**
This pull request addresses the following links:
- https://github.com/episphere/connect/issues/1063

Requests from issue:
- change query to get eligible home mouthwash kit participants using `desc` order (newest first)
- create 5 day lag for eligible home mouthwash kit participants to appear in query (Exclude eligible participants with `bloodOrUrineCollectedTimestamp` (740582332) within range from today's current date to 5 days before today's current date )
- Filter out P.O. Box in Address Line 1 (521824358)

Code Changes: 
- added a 'desc' argument to change OrderBy clause
- created `getFiveDaysAgoDateISO ` function and added another `where` clause to remove eligible participants with a bloodOrUrineCollectedTimestamp of five days or less than the current date.
- added [regex](https://github.com/episphere/connectFaas/compare/dev...issue%231063-sort-eligiblePt-desc-and-five-day-lag?expand=1#diff-be91e66430aa4dd2f61668d1bd17d3cb449f57947e13bfa2f6bc087919967282R2285) to test for a "p.o. box match" in Address Line 1 (521824358). While snapshot is iterated and mapped, any regex of true will be returned as `null` and filtered out of array.

Note to self:  Request new composite index for the different environment (stage & prod)

### CODE REVIEW

**Review by jhflorey — APPROVED** [review]

File: utils/firestore.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const db = admin.firestore();
 const { tubeConceptIds, collectionIdConversion, swapObjKeysAndValues, batchLimit, listOfCollectionsRelatedToDataDestruction, createChunkArray, twilioErrorMessages, cidToLangMapper, printDocsCount } = require('./shared');
 const fieldMapping = require('./fieldToConceptIdMapping');
 const { isIsoDate } = require('./validation');
+const { getFiveDaysAgoDateISO } = require('./shared');
```
Comment (we-ai) [inline utils/firestore.js:9]: Merge this to line 6.

File: utils/firestore.js [hunk 2]
```diff
@@ -2270,16 +2280,22 @@ const addKitStatusToParticipant = async (participantsCID) => {
...
     const processedRecord = {
-        first_name: record[firstName],
-        last_name: record[lastName],
-        address_1: record[address1],
-        address_2: record[address2] || '',
-        city: record[city],
-        state: record[state],
-        zip_code: record[zip], 
-        connect_id: record['Connect_ID'],
+    first_name: record[firstName],
```
Comment (we-ai) [inline utils/firestore.js:2291]: Use JS conventions: no snake cases if you happen to adjust them.

**Review by we-ai — APPROVED** [review]
Looks good. Minor comments.

File: utils/firestore.js (hunk 2 above)
Comment (Gbarra9) [reply utils/firestore.js:2291]: This is in snake case because it goes through the csv reports.

**Pull request merged**